arraycontext/context.py

```diff
@@ -463,6 +464,15 @@ def compile(self, f: Callable[..., Any]) -> Callable[..., Any]:
         """
         return f
 
+    def force_evaluate(self, array: Array) -> Array:
+        """Force the evaluation of any expression trees in the input *array* and
+        return a 'thaw'ed version of that array.  This function forces evaluation
+        by calling 'freeze' and subsequently 'thaw'.
```

Comment:
You can talk about evaluation of expression trees as an illustrative example, but not when you define what this does. Think of the documentation triangle: what, how, why. Keep them separate, and in that order.